program/tests/create_lookup_table_ix.rs

```diff
@@ -1,147 +1,168 @@
+    assert_eq!(
+        lookup_table_account.lamports(),
+        Rent::default().minimum_balance(LOOKUP_TABLE_META_SIZE)
+    );
+    let lookup_table = AddressLookupTable::deserialize(&lookup_table_account.data()).unwrap();
+    assert_eq!(lookup_table.meta.deactivation_slot, Slot::MAX);
+    assert_eq!(lookup_table.meta.authority, Some(authority));
+    assert_eq!(lookup_table.meta.last_extended_slot, 0);
+    assert_eq!(lookup_table.meta.last_extended_slot_start_index, 0);
+    assert_eq!(lookup_table.addresses.len(), 0);
 
     // Second create should succeed too
-    {
-        let recent_blockhash = client
-            .get_new_latest_blockhash(&recent_blockhash)
-            .await
-            .unwrap();
-        let transaction = Transaction::new_signed_with_payer(
-            &[create_lookup_table_ix],
-            Some(&payer.pubkey()),
-            &[payer],
-            recent_blockhash,
-        );
-
-        assert!(matches!(
-            client.process_transaction(transaction).await,
-            Ok(())
-        ));
-    }
+    mollusk.process_and_validate_instruction(
+        &create_lookup_table_ix,
+        &[
+            (lookup_table_address, lookup_table_account.clone()),
+            (authority, AccountSharedData::default()),
+            (payer, AccountSharedData::default()), // Note the lack of lamports.
+            keyed_account_for_system_program(),
+        ],
+        &[Check::success()],
+    );
 }
 
-#[tokio::test]
-async fn test_create_lookup_table_use_payer_as_authority() {
-    let mut context = setup_test_context().await;
+#[test]
+fn test_create_lookup_table_use_payer_as_authority() {
+    let mut mollusk = setup();
 
     let test_recent_slot = 123;
-    overwrite_slot_hashes_with_slots(&context, &[test_recent_slot]);
-
-    let client = &mut context.banks_client;
-    let payer = &context.payer;
-    let recent_blockhash = context.last_blockhash;
-    let authority_address = payer.pubkey();
-    let transaction = Transaction::new_signed_with_payer(
-        &[create_lookup_table(authority_address, payer.pubkey(), test_recent_slot).0],
-        Some(&payer.pubkey()),
-        &[payer],
-        recent_blockhash,
-    );
 
-    assert!(matches!(
-        client.process_transaction(transaction).await,
-        Ok(())
-    ));
-}
+    // [Core BPF]: Warping to slot, which will update `SlotHashes`.
+    mollusk.warp_to_slot(test_recent_slot + 1);
 
-#[tokio::test]
-async fn test_create_lookup_table_not_recent_slot() {
-    let mut context = setup_test_context().await;
-    let payer = &context.payer;
-    let authority_address = Pubkey::new_unique();
-
-    let ix = create_lookup_table(authority_address, payer.pubkey(), Slot::MAX).0;
-
-    assert_ix_error(
-        &mut context,
-        ix,
-        None,
-        InstructionError::InvalidInstructionData,
-    )
-    .await;
+    let payer = Pubkey::new_unique();
+    let authority = Pubkey::new_unique();
```

Comment:
This test appears to be doing something different now -- authority should be the same as payer, right?

Reply:
Updated. Turns out I had to fix a little bug in Mollusk's transaction deduping (hence the version bump). All set now.